SoomlaiOSCore/data/storages/KeyValueDatabase+DatabaseSelector.m

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Eugene Butusov on 25/12/15.
+// Copyright (c) 2015 SOOMLA. All rights reserved.
+//
+
+#import "KeyValueDatabase+DatabaseSelector.h"
+#import "SQLiteDatabase.h"
+#import "UserDefaultsDatabase.h"
+
+
+@implementation KeyValueDatabase (DatabaseSelector)
+
++(KeyValueDatabase *)databaseByType:(KeyValueDatabaseType)databaseType {
```

Comment:
@vedi, in ObjC factory method names starts with type that will be spawn, for example: https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSDictionary_Class/#//apple_ref/doc/uid/20000140-SW10